fractal_server/tasks/collection.py

```diff
@@ -483,6 +483,25 @@ async def _pip_install(
         cwd=venv_path, command=cmd_install, logger_name=logger_name
     )
 
+    if task_pkg.pinned_package_versions:
+        for pkg_name, pkg_version in task_pkg.pinned_package_versions.items():
```

Comment:
Maybe we can use `pinned_pkg_{name,version}`, to avoid confusion with `task_pkg.package_version` (also present in this function).